docs/framework/winforms/windows-forms-accessibility-improvements.md

```diff
@@ -0,0 +1,96 @@
+---
+title: "Windows Forms Accessibility Improvements"
+ms.date: "08/16/2019"
+helpviewer_keywords: 
+  - "Windows Forms accessibility"
+  - "accessibility"
+ms.assetid: 
+---
+# Accessibility improvements in Windows Forms controls for .NET Core
+
+## Scope
+Major
+
+## Version Introduced
+.NET Core 3.0
+
+## Change Description
+The Windows Forms is continuing to improve how it works with accessibility technologies to better support Windows Forms customers. These include the following changes:
+- Changes in various areas of interaction with accessibility client applications including Narrator.
+- Changes in the Accessible hierarchy (improving navigation through the UI Automation tree).
+- Changes in keyboard navigation.
+
+- [x] The quirks which were used in .NET Framework 4.7.1 - 4.8 to opt-in/out new accessibility features have been removed, so previously quirked behavior is now enabled by default.
+
+### Recommended Action
+__The changes which could be opted in or out in .NET Framework 4.7.1 - 4.8 are now enabled by default and cannot be opted out__
+  
+Applications now benefit from all the new accessibility features (introduced in .NET Framework 4.7.1 - 4.8) without additional configuration.
+
+__Improved ListBox Accessibility support__
```

Comment:
Each of these should be a level-2 head (\#\#)

Reply:
Fixed, thanks.